streamer/packager_node.py

```diff
@@ -144,6 +144,9 @@ def _setup_stream(self, stream: OutputStream) -> str:
     if stream.input.drm_label:
       dict['drm_label'] = stream.input.drm_label
 
+    if stream.type == MediaType.TEXT and stream.forced_subtitle:
```

Comment:
and instead here we could use `stream.input.forced_subtitles` to query this value.

Reply:
Done!